### PR TITLE
Prepare release 1.8

### PR DESCRIPTION
**Pull request by wz1000**
- Prepare 1.8.0.0
- fix ci
- Bump hie-compat
- Use hiedb from hackage

<a href="https://gitpod.io/#https://github.com/haskell/haskell-language-server/pull/3150"></a>

### CODE REVIEW

**Review by konn — APPROVED** [review]
LGTM as for the Splice Plugin. Thanks!

**Comment by santiweight** [comment]
What's the timeline on this release? @isovector and I are close to getting Wingman up and running in this release. 

At the very least we have empty-case completion working, so it'd be good to get that in before HLS 1.8...

**Review by kokobd — APPROVED** [review]

**Comment by kokobd** [comment]
And we also need to release `lsp`, `lsp-types`, and `lsp-test`

**Review by fendor — APPROVED** [review]
LGTM, not much else for me to say :)

File: plugins/hls-tactics-plugin/hls-tactics-plugin.cabal [hunk 1]
```diff
@@ -1,7 +1,7 @@
 cabal-version:      2.4
 category:           Development
 name:               hls-tactics-plugin
-version:            1.6.2.0
```
Comment (July541) [inline plugins/hls-tactics-plugin/hls-tactics-plugin.cabal:4]: why 1.8?

**Review by July541 — APPROVED** [review]
I noticed some plugins are modified without version bumps, any reason here?

**Review by drsooch — APPROVED** [review]

**Comment by wz1000** [comment]
> I noticed some plugins are modified without version bumps, any reason here?

Plugins which haven't had a release since their last version bump and plugins which have had minor changes to their cabal files which can be accommodated with a hackage revision haven't been bumped.

File: plugins/hls-tactics-plugin/hls-tactics-plugin.cabal (hunk 1 above)
Comment (wz1000) [reply plugins/hls-tactics-plugin/hls-tactics-plugin.cabal:4]: Changed to 1.7.

File: ChangeLog.md [hunk 2]
```diff
@@ -1,5 +1,266 @@
...
 
+## 1.8.0.0
+
+- Binaries for GHC 9.2.3 and GHC 9.2.4
+- Initial support for GHC 9.4 with binaries for GHC 9.4.1 and GHC 9.4.2
+- Startup time and performance improvements on projects using Template Haskell by serializing intermediate core (#2813)
+- Improvements for hls-class-plugin (#2920, @July541) 
+- The new hls-gadt-plugin (#2899, @July541)
+- Moving code actions from ghcide to the new hls-refactor-plugin (#3091, @wz1000)
+- Many more improvements and bug fixes thanks to our contributors!
```
Comment (pepeiborra) [inline ChangeLog.md:11]: Worth highlighting the memory savings achieved by @kokobd migrating  file paths to a compact representation

**Review by pepeiborra — APPROVED** [review]
Thanks!

**Review by hasufell — APPROVED** [review]
Did you trigger gitlab CI test run?

**Comment by wz1000** [comment]
> What's the timeline on this release? @isovector and I are close to getting Wingman up and running in this release.

I hope to release tomorrow. Given that we are already late, I am reluctant to delay the release much further.

We aim to restore a more frequent release cycle now with monthly or bi-monthly releases, so if your patch doesn't make it into this release, the next one shouldn't be as far off as 1.8 was from 1.7.

**Pull request merged**